README.md

```diff
@@ -25,113 +30,96 @@ Build a GraphQL API that can `create/read/update/delete` user data from a persis
+- [x] GraphQL API documentation
+- [x] Ensure your API is able to support all requirements passed to the consumer team
 
 ### Bonus
 
 These may be used for further challenges. You can freely skip these; feel free to try out if you feel up to it.
 
 #### Developer Experience (in order)
 
-1. E2E Testing
-1. Integration testing
-1. Code-coverage report generation
-1. Describe your strategy for Lambda error handling, retries, and DLQs
-1. Describe your cloud-native logging, monitoring, and alarming strategy across all queries/mutations
-1. Online interactive demo with a publicly accessible link to your API
-1. Brief description of the frameworks/tools used in the solution
-1. Optimized lambda build.
-1. Commit linting
-1. Semantic release
+- [ ] E2E Testing
+	- none
+- [ ] Integration testing
+	- none
+- [x] Code-coverage report generation
+	- not complete, but generated
+- [x] Describe your strategy for Lambda error handling, retries, and DLQs
+- [x] Describe your cloud-native logging, monitoring, and alarming strategy across all queries/mutations
+- [x] Online interactive demo with a publicly accessible link to your API
+	- the insomnia queries collection??
+- [x] Brief description of the frameworks/tools used in the solution
+- [x] Optimized lambda build
+	- lambda layer is composed of the solution's git package javascript version
+- [x] Commit linting
+- [ ] Semantic release
 
 
 #### API Consumer Experience (in order)
 
-1. Document how consumers can quickly prototype against your APIs
-    - GraphQL Playground setup
-    - Insomnia setup
-    - Feel free to use any other tool/client you might know that enable consumers to prototype against your API
-1. GraphQL Documentation Generation
-1. Client API generation
+- [x] Document how consumers can quickly prototype against your APIs
+    - [ ] GraphQL Playground setup
+    - [x] Insomnia setup
+    - [ ] Feel free to use any other tool/client you might know that enable consumers to prototype against your API
+- [x] GraphQL Documentation Generation
+	- also in the DOCUMENTATION folder, done with https://app.graphqleditor.com/
+- [x] Client API generation
 
 
 ## Client context
 
-Assume the GraphQL API you are developing will be used by a hypothetical front-end team to build the following screens:
-
-![Superformula-front-end-test-mockup](./mockup1.png)
-
-![Superformula-front-end-test-mockup-2](./mockup2.png)
-
-> [Source Figma file](https://www.figma.com/file/hd7EgdTxJs2fpTzzSKlNxo/Superformula-full-stack-test)
-
 - Client will be performing real-time search against this API
+	- **MY MAIN CONCERN**, I think that the thing that really sells my point was the DynamoDb speed over size. With the replication that was implemented, we can make sure that all queries consumes the least amount of read capacity
+	- The write capacity I'd spreadout with a queue service, so we dont burst the read capacity quota
 - List of users should be updated automatically after single user is updated
```

Comment:
How would you instruct a front-end engineer to implement this feature?

Reply:
The front-end engineer would need to make a post, graphql request, to the listUsers endpoint.
He/She would have access to the graphql schema, so, unless it's a jr engineer, it would be a pretty straighforward call